test/workers/delete_object_hierarchy_worker_test.rb

```diff
@@ -177,19 +177,20 @@ def setup
     end
   end
 
-  class DeleteMemberPermissionTest < DeleteObjectHierarchyWorkerTest
+  class DeleteMemberPermissionThroughUserTest < ActiveSupport::TestCase
+    def setup
+      tenant = FactoryBot.create(:simple_provider)
+      @member = FactoryBot.create(:member, account: tenant)
+      @member_permission = FactoryBot.create(:member_permission, user: member)
+    end
 
-    def test_member_permission
```

Comment:
Correct me if I am wrong but I have the perception that this test wasn't really testing anything at all 😕